telegram/_business.py

```diff
@@ -453,3 +455,69 @@ def de_json(
         )
 
         return super().de_json(data=data, bot=bot)
+
+    def get_opening_hours_for_day(
+        self, target_date: dt, tzinfo: Optional[zoneinfo.ZoneInfo] = None
+    ) -> Tuple[Tuple[datetime, datetime], ...]:
+        """
+        Get the opening hours for a specific day.
+
+        .. versionadded:: NEXT.VERSION
+
+        Args:
+            target_date (:class:`datetime.date`): The date for which to get the opening hours.
+            tzinfo (:class:`zoneinfo.ZoneInfo`, optional): The timezone to use for opening hours.
+                If :obj:`None`, the time zone of the business is used, i.e. :attr:`time_zone_name`.
+                Defaults to :obj:`None`.
+
+        Returns:
+            Tuple[Tuple[:class:`datetime.datetime`, :class:`datetime.datetime`], ...]:
+                A tuple of tuples containing the opening and closing times for the day.
+        """
+        if tzinfo is None:
+            tzinfo = zoneinfo.ZoneInfo(self.time_zone_name)
```

Comment:
https://docs.python.org/3.11/library/zoneinfo.html#data-sources

example error:

```python
zoneinfo._common.ZoneInfoNotFoundError: 'No time zone found with key Europe/Berlin'
```

IMO we should handle this a bit better and tell the user that they should install `tzdata` in that case. I recommend  using try-except and reraising a `ZoneInfoNotFoundError` with a more descriptive error message. It will be good to provide a small wrapper function for that in `telegram/_utils/datetime.py` so that it can be reused (see also line 509)